ckanext/datastore/backend/postgres.py

```diff
@@ -85,6 +85,14 @@
 _UPSERT = 'upsert'
 _UPDATE = 'update'
 
+_OPERATORS = {
+    'eq': '=',
+    'gt': '>',
+    'gte': '>=',
+    'lt': '<',
+    'lte': '<=',
+    'in': 'in',
```

Comment:
let's drop `'in'` because it's not required for pagination and we don't have tests covering its edge cases here